prov/efa/src/efa_prov.c

```diff
@@ -65,7 +65,12 @@ int efa_win_lib_initialize(void)
 	* efa_load_efawin_lib function will replace stub ibv_* functions with
 	* functions from efawin dll
 	*/
-	return efa_load_efawin_lib();
+	int err = efa_load_efawin_lib();
+	if (ret) {
```

Comment:
`ret` should have been `err`

Reply:
Done, I switched from ret to err as a last minute change and didn't check to recompile *facepalm*